server/routerlicious/packages/services-utils/src/executeOnInterval.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Lumberjack } from "@fluidframework/server-services-telemetry";
+
+export async function executeOnInterval<T>(
+    api: () => Promise<T>,
+    intervalInMs: number,
+    callName: string): Promise<void> {
+    await api()
```

Comment:
In order to avoid a caller accidentally awaiting an infinite loop, maybe this `await` should be removed, and the function type could just be `export function executeOnInterval(...) void {}`.

Currently it would be fine, as it would just await the first call to `api`, but if an `await` were added on line 25, then `await executeOnInterval()` would freeze the thread. If there is no need to await the first api call, then I think we should just fire-and-forget here.